pkg/sdk/database_role_test.go

```diff
@@ -2,10 +2,12 @@ package sdk
 
 import (
 	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/random"
 )
 
 func TestDatabaseRoleCreate(t *testing.T) {
-	id := randomDatabaseObjectIdentifier(t)
+	id := RandomDatabaseObjectIdentifier(t)
```

Comment:
shouldnt this be random.DatabaseObjectIdentifier()?

Reply:
I started doing this earlier, but:
- these `RandomXIdentifier(t)` are using underneath the `random.Y(t)`; it means that I would have to rework also `random.Y(t)` ones
- some `random.Y(t)` on the other hand are using `require.NoError(t, err)` which require this `t` param.

But I agree that having them without the `t` part is better, so I will rework them.

Reply:
Done.